t/admin/routes.t

```diff
@@ -705,20 +705,17 @@ GET /t
                         "uri": "/index.html"
                 }]],
                 [[{
-                    "node": {
```

Comment:
we'd better keep the old response structure.

Reply:
Hi @membphis! Response of etcd v3 is quite different from v2. For example like this, v3 would not return `action`, and flatten "node-nodes" structure into one layer "kvs". Thus I try to only return content inside node/nodes and kvs, to develop compatibility of different etcd version,

But modifying all test files is quite a big work. So I just submit a PR to mock etcd v2 response in #2036, keeping the old structure if we need. 
But I still believe it could be better to get closer to v3 structure, since v2 could be deprecated one day, and there are some new functions of v3 that could not been mocked into v2.